data/mods/Magiclysm/itemgroups/itemgroups.json

```diff
@@ -292,19 +292,18 @@
     "id": "enchanted_misc",
     "type": "item_group",
     "//": "all enchanted miscellanious items",
-    "subtype": "collection",
     "items": [
-      { "item": "heat_cube", "prob": 100 },
-      { "item": "mkey_opening", "prob": 100 },
-      { "item": "mtorch_everburning", "prob": 100 },
-      { "item": "mflask_hip_whiskey", "prob": 100 },
-      { "item": "mtailors_kit", "prob": 100 },
-      { "item": "mspider_box", "prob": 100 },
-      { "item": "cauldron_demon_chitin", "prob": 20 },
-      { "item": "fridge_holding_1", "prob": 12 },
-      { "item": "bag_holding_1", "prob": 8 },
-      { "item": "bag_holding_2", "prob": 1 },
-      { "item": "cauldron_orichalcum", "prob": 100 }
+      [ "heat_cube", 100 ],
+      [ "mkey_opening", 100 ],
+      [ "mtorch_everburning", 100 ],
+      [ "mflask_hip_whiskey", 100 ],
+      [ "mtailors_kit", 100 ],
+      [ "mspider_box", 100 ],
+      [ "cauldron_demon_chitin", 20 ],
+      [ "fridge_holding_1", 12 ],
+      [ "bag_holding_1", 8 ],
+      [ "bag_holding_2", 1 ],
+      [ "cauldron_orichalcum", 100 ]
```

Comment:
i'm willing to bet this is totally unecessary. can you revert this part with the removal of collection and check that it still does it in game for me? i want to reduce unecessary churn, and i prefer the object way of representing things in json.

Reply:
So revert the changes to the item list and only remove the ‘“subtype”: “collection”’?